### DISCUSSION TITLE
How to enable HTTPS support for the HTTP request block in Dify.

### DISCUSSION THREAD

**Discussion by ismasher2000** [langgenius/dify#9480] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

My API backend must be based on HTTPS for security compliance purposes, but I cannot find a way to enable HTTPS support for the HTTP Request Block. After testing, it seems that the HTTP Request Block does not support HTTPS at all, which poses certain security risks.

### 2. Additional context or comments

When connecting to my HTTPS backend API using HTTPS in HTTP Request Block, I get a 403 Forbidden error. If I connect to my HTTPS backend using HTTP  in HTTP Request Block, the status is successful, but the return message says: "The plain HTTP request was sent to HTTPS port."

### 3. Can you help us with this feature?

- [ ] I am interested in contributing to this feature.

**Comment by zhangdahai112** [comment]
i wrote a project to solve this by forwarding http request to unlimited server 
https://github.com/zhangdahai112/DifyHttpsProxy
